docs/tutorial.md

```diff
@@ -120,58 +156,46 @@ time $PASH_TOP/pa.sh $PASH_TOP/evaluation/intro/hello-world.sh
 
 #### A More Interesting Script: Demo Spell
 
-We will use `demo-spell.sh` --- a pipeline based [on the original Unix spell program](https://dl.acm.org/doi/10.1145/3532.315102) by Johnson --- to confirm that the infrastructure works as expected.
-
-First, we need to setup the appropriate input files for this script to execute:
-
+First, `cd` to `$PASH_TOP/evaluation/intro`:
```

Comment:
Wait, isn't demo-spell very similar log analysis that was shown above?

Reply:
That part is just an intro, it doesn't include any running, but we can discuss how we want to structure it.